### PR TITLE
[MAPSIOS-449] update core styles

### PR DESCRIPTION
**Pull request by ank27**
Update core styles to latest version:

MAPBOX_STREETS -> mapbox://styles/mapbox/streets-v12
SATELLITE_STREETS -> mapbox://styles/mapbox/satellite-streets-v12
OUTDOORS -> mapbox://styles/mapbox/outdoors-v12
LIGHT -> mapbox://styles/mapbox/light-v11
DARK -> mapbox://styles/mapbox/dark-v11

## Pull request checklist:
 - [ ] Describe the changes in this PR, especially public API changes.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. Put tests in correct [Test Plan](https://github.com/mapbox/mapbox-maps-ios/tree/main/Tests/TestPlans) (Unit, Integration, All)
   - [ ] If tests were not written, please explain why.
 - [ ] Add documentation comments for any added or updated public APIs.
 - [ ] Add any new public, top-level symbols to the Jazzy config's `custom_categories` (scripts/doc-generation/.jazzy.yaml)
 - [ ] Add a changelog entry to to bottom of the relevant section (typically the `## main` heading near the top).
 - [ ] Update the guides (internal access only), README.md, and DEVELOPING.md if their contents are impacted by these changes.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` first and then port to `v10.[version]` release branch.

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-ios/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Review by maios — APPROVED** [review]

File: Sources/MapboxMaps/Style/StyleURI.swift [hunk 1]
```diff
@@ -29,21 +29,21 @@ public struct StyleURI: Hashable, RawRepresentable {
     }
 
     /// Mapbox Streets is a general-purpose style with detailed road and transit networks.
-    public static let streets = StyleURI(rawValue: "mapbox://styles/mapbox/streets-v11")!
+    public static let streets = StyleURI(rawValue: "mapbox://styles/mapbox/streets-v12")!
```
Comment (evil159) [inline Sources/MapboxMaps/Style/StyleURI.swift:32]: It seems that globe is enabled by default in new styles - is this expected?

**Review by evil159 — APPROVED** [review]
New styles look 🔥 There seem to be `testAttributionFeedbackURL` test in `AttributionTests` failing because it still expects `streets-v11` to be present in the feedback URL.

File: Sources/MapboxMaps/Style/StyleURI.swift (hunk 1 above)
Comment (ank27) [reply Sources/MapboxMaps/Style/StyleURI.swift:32]: yes its default behaviour for v12 styles.

**Comment by ank27** [comment]
> There seem to be `testAttributionFeedbackURL` test in `AttributionTests` failing because it still expects `streets-v11` to be present in the feedback URL.

thanks for catching, updated that 👍

**Comment by OdNairy** [comment]
Closed in favor of 1496 PR

**Pull request closed without merging**